ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -23,7 +27,11 @@ interface WorkflowsRowProps {
 export function WorkflowsRow(props: WorkflowsRowProps) {
     const [hideDrawer, setHideDrawer] = useState(true);
     const wf = props.workflow;
-    const workflowLink = uiUrl(`workflows/${wf.metadata.namespace}/${wf.metadata.name}?uid=${wf.metadata.uid}`);
+    // title + description vars
```

Comment:
Can we remove? Wdyt?

Reply:
I added it bc we have some long lists of vars elsewhere that have some comments. A comment makes those lists a little easier on the eyes with some visual separation / categorization.

Reply:
I'd prefer a newline over a redundant comment.

Reply:
I'd rather specify a category 🤷 I don't think it's entirely redundant either given the last two variables are different by name at least (from a quick glance I pay attention to the names much more than the logic)

Reply:
👎 is that I disagree but not enough to block this PR. Maybe I can learn something from this perspective. 🤷

Reply:
honestly I can't say I care too much about it either, but enough to prefer it 😅